Bug: latest typescript: missing `altitudeAngle`, `azimuthAngle` in `FederatedPointerEvent` (incorrectly implements 'PointerEvent')

### Current Behavior

type error: 

```
node_modules/pixi.js/lib/events/FederatedPointerEvent.d.ts:6:22 - error TS2420: Class 'FederatedPointerEvent' incorrectly implements interface 'PointerEvent'.
  Type 'FederatedPointerEvent' is missing the following properties from type 'PointerEvent': altitudeAngle, azimuthAngle
```

### Expected Behavior

no type error

### Steps to Reproduce

update to latest pixi, latest typescript

### Environment

- **`pixi.js` version**: *pixi.js@8.6.4* (latest as of 2024-12-10)
- **`typescript` **: *typescript@5.7.2* (latest as of 2024-12-10)
- **Browser & Version**: *electron@32.1.2*
- **OS & Version**: *e.g. macos m1, latest version*
- **Running Example**: *e.g. <https://pixiplayground.com/>*


### Possible Solution

add the two missing properties somewhere so they end up in in node_modules/pixi.js/lib/events/FederatedPointerEvent.d.ts

```
    altitudeAngle: number;
    azimuthAngle: number;
```

### Additional Information

_No response_

--- a/src/events/FederatedPointerEvent.ts
+++ b/src/events/FederatedPointerEvent.ts
@@ -20,6 +20,22 @@ export class FederatedPointerEvent extends FederatedMouseEvent implements Pointe
     public width = 0;
 
     /**
+     * The angle in radians of a pointer or stylus measuring the vertical angle between
+     * the device's surface to the pointer or stylus.
+     * A stylus at 0 degrees would be directly parallel whereas at π/2 degrees it would be perpendicular.
+     * @see https://developer.mozilla.org/docs/Web/API/PointerEvent/altitudeAngle)
+     */
+    public altitudeAngle: number;
+
+    /**
+     * The angle in radians of a pointer or stylus measuring an arc from the X axis of the device to
+     * the pointer or stylus projected onto the screen's plane.
+     * A stylus at 0 degrees would be pointing to the "0 o'clock" whereas at π/2 degrees it would be pointing at "6 o'clock".
+     * @see https://developer.mozilla.org/docs/Web/API/PointerEvent/azimuthAngle)
+     */
+    public azimuthAngle: number;
+
+    /**
      * The height of the pointer's contact along the y-axis, measured in CSS pixels.
      * radiusY of TouchEvents will be represented by this value.
      * @see https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/height

